articles/stream-analytics/stream-analytics-stream-analytics-query-patterns.md

```diff
@@ -528,113 +636,39 @@ Use **LAG** to view the input stream for 24 hours and look for instances where *
 | 2014-01-01T14:01:40.000Z |2014-01-01T14:01:35.000Z |6 |
 | 2014-01-01T14:01:45.000Z |2014-01-01T14:01:35.000Z |6 |
 
-**Solution**:
+**Query**:
 
 ```SQL
-    SELECT
-        System.Timestamp() AS windowEnd,
-        TopOne() OVER (ORDER BY t DESC) AS lastEvent
-    FROM
-        input TIMESTAMP BY t
-    GROUP BY HOPPINGWINDOW(second, 300, 5)
+SELECT
+	System.Timestamp() AS Window_end,
+	TopOne() OVER (ORDER BY Time DESC) AS Last_event
+FROM
+	Input TIMESTAMP BY Time
+GROUP BY
+	HOPPINGWINDOW(second, 300, 5)
 ```
 
-**Explanation**:
-This query generates events every 5 seconds and outputs the last event that was received previously. The [Hopping window](/stream-analytics-query/hopping-window-azure-stream-analytics) duration determines how far back the query looks to find the latest event (300 seconds in this example).
+This query generates events every 5 seconds and outputs the last event that was received previously. The **HOPPINGWINDOW** duration determines how far back the query looks to find the latest event.
 
+For more information, refer to [Hopping window](/stream-analytics-query/hopping-window-azure-stream-analytics).
 
-## Query example: Correlate two event types within the same stream
+## Query example: Process events independent of Device Clock Skew (Substreams)
```

Comment:
it seems substreams is more than clock skew. What about this title: "Process events using independent device timeline (substreams)